integreat_cms/cms/views/bulk_action_views.py

```diff
@@ -278,14 +278,14 @@ def post(self, request, *args, **kwargs):
                     messages.error(
                         request,
                         _(
-                            "{} cannot be archived because it was embedded as live content from another page."
-                        ).format(content_object.backend_translation.title),
+                            'Page "{}" cannot be archived because it was embedded as live content from another page.'
+                        ).format(content_object.default_translation),
```

Comment:
```suggestion
                        ).format(content_object.best_translation),
```
Normally, we use the property `best_translation` in such cases - the backend translation if it exists, the default translation otherwise.